gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcReadChannel.java

```diff
@@ -124,6 +122,9 @@
   // Offset in the object for the end of the range-requests
   private long contentChannelEndOffset = -1;
 
+  // Read throughput on a single call used to estimate the timeouts
+  private static final long ESTIMATED_OBJECT_READ_SPEED_PER_SEC = 50 * 1024 * 1024; // 50 MBps
```

Comment:
should this also be a configuration instead of hard coding?

Reply:
Makes sense